### ISSUE TITLE
Special characters in a certain source 

### ISSUE THREAD

**Issue by Mingo3347** [LNReader/lnreader#1027]
### Steps to reproduce

Open any novel in (فضاء الروايات) source and the chapters contains tons of special characters between every single paragraph making it very annoying........this didn't exist in the previous version of the app but only after I updated to the beta version of 2.0.0
![Screenshot_20240407-043850_LNReader](https://github.com/LNReader/lnreader/assets/108782331/94fc5166-2b4b-4e9c-a88d-88916c9670b5)

### Expected behavior

Only the chapters and the content of the novel should be visible 

### Actual behavior

Special characters 

### LNReader version

2.0.0

### Android version

Android 11

### Device

Samsung a70 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[2.0.0](https://github.com/lnreader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by rajarsheechatterjee** [comment]
> If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
